clang/docs/UsersManual.rst

```diff
@@ -4401,13 +4415,21 @@ Execute ``clang-cl /?`` to see a list of supported options:
                               Instrument only functions from files where names don't match all the regexes separated by a semi-colon
       -fprofile-filter-files=<value>
                               Instrument only functions from files where names match any regex separated by a semi-colon
-      -fprofile-instr-generate=<file>
-                              Generate instrumented code to collect execution counts into <file>
+      -fprofile-generate=<dirname>
+                              Generate instrumented code to collect execution counts into a raw profile file in the directory specified by the argument. The filename uses the %m format. See :ref:`Profiling With Instrumentation <prof_instr>`  section for details.
+                              (overridden by LLVM_PROFILE_FILE env var)
+      -fprofile-generate
+                              Generate instrumented code to collect execution counts into default_%m.profraw file
+                              (overridden by '=' form of option or LLVM_PROFILE_FILE env var)
+      -fprofile-instr-generate=<file_name_pattern>
```

Comment:
Can you also add `-fprofile-instr-generate` case below for consistency?

Reply:
done (-fprofile-instr-generate was there before. I added -fprofile-generate).